### PR TITLE
Breakout CSS and change it to be opt-in

### LINKED ISSUE #9: all non required styles should be opt in
similar to ember-modal-dialog. fixed column styles and the overflow hidden on columns should likely be included, but the rest should be opt in stylesheet.

### PR DESCRIPTION
**Pull request by pgrippi**
- Changes all CSS to be opt-in
- Break the CSS into two files: structure and appearance. Structure should define the bare-minimum CSS needed such that only it can be included for the table to work correctly. Appearance should define a custom look and feel. With that in mind, I'm open to moving definitions around as some could fall into either category.

One other change in here is the addition of the `ember-truth-helpers` dependency. In the [table-columns.hbs:18](https://github.com/cball/justa-table/blob/master/addon/templates/components/table-columns.hbs#L18) template the `eq` helper is used, without ember-truth-helpers it breaks the app.

Should close #9 if this is what you had in mind

This was made to fix a bug where I was forced to use `!important` in my CSS to change the background color of a row because it was [set by the CSS](https://github.com/cball/justa-table/blob/master/addon/styles/_justa-table.scss#L61) in this project.

### CODE REVIEW

**Comment by cball** [comment]
@pgrippi thanks! this is exactly what I wanted to do.

File: README.md [hunk 1]
```diff
@@ -2,11 +2,24 @@
 
 This README outlines the details of collaborating on this Ember addon.
 
-## Usage
+## Including In An Ember Application
+
+Here is the simplest way to get started with ember-modal-dialog:
```
Comment (cball) [inline README.md]: should probably change the end of this line. :smile:

**Comment by cball** [comment]
One other minor thing, can you name the `scss` files with underscores since they are meant to be imported into another stylesheet?

File: README.md (hunk 1 above)
Comment (pgrippi) [inline README.md]: oh crap

**Comment by pgrippi** [comment]
> can you name the `scss` files with underscores

I've updated the file names to start with an underscore, which is what I think you were saying. Did you want me to replace the hyphens with underscores too?

Also I fixed the README, I think I got a bit overzealous with my copy-and-pasting...

**Comment by cball** [comment]
Sorry, pre :coffee: comment. prefix with underscores is what I was saying.

**Pull request merged**